theme/templates/resource-landing-page/modals.html

```diff
@@ -475,7 +475,8 @@ <h4 class="modal-title" id="submit-for-publication-title">Are you sure you want
             </div>
             <div class="modal-footer">
                 <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
-                <a type="button" id="publish-btn-2" class="btn btn-danger" href="/hsapi/_internal/{{ cm.short_id }}/publish/">Publish</a>
+                <a type="button" id="publish-btn-2" class="btn btn-danger" href="/hsapi/_internal/{{ cm.short_id }}/publish/doi">Publish with DOI</a>
+                <a type="button" id="publish-btn-2" class="btn btn-danger" href="/hsapi/_internal/{{ cm.short_id }}/publish/minid">Publish with MINID</a>
             </div>
```

Comment:
Noticed two buttons have the same id ```publish-btn-2``` which would give us issues. Should give two buttons unique different ids.